styles/styles.css

```diff
@@ -0,0 +1,370 @@
+    justify-content: center;
+}
+
+.inner-container {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 900px;
+    justify-content: flex-start;
+    align-items: center;
+}
+
+.aside {
+    flex-basis: 1000px;
+    flex-shrink: 3;
+}
+
+/* ------------------ BONUS OPDRACHT ------------------- */
+
+.triangle {
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 40px 0 0 100vw;
+    border-color: transparent transparent transparent var(--wit);
+    margin-top: -40px;
+}
+
+.background-words {
+    position: absolute;
+    color: var(--lichtpaars);
+    z-index: -1;
+    font-size: calc(100vw / 5);
+}
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+
+header {
+    color: #fff;
+    background-color: var(--paars);
+    background-image: url("../assets/noise-texture.png"); /* BONUS OPDRACHT */
+}
+
+nav {
+    display: flex;
+    width: 100%;
+    flex-direction: row;
+    justify-content: flex-end;
+}
+
+nav ul {
+    display: flex;
+    list-style-type: none;
+    margin-right: 20px;
+    gap: 20px;
+}
+
+nav li {
+    padding: 10px;
+}
+
+.header {
+    text-align: center;
+    padding: 2rem;
+}
+
+header button {
+    color: #fff;
+    padding: 15px;
+    margin: 30px;
+    background-color: var(--rose);
+    border-radius: 3rem;
+    text-align: center;
+}
+
+.inner-teaser p {
+    padding-top: 3rem;
+}
+
+.stripe {
+    width: 50px;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+
+.inner-intro {
+    padding: 70px;
+    text-align: center;
+}
+
+.inner-intro h3 {
+    color: var(--paars);
+    padding: 10px;
+}
+
+.inner-intro p {
+    padding-top: 40px;
+}
+
+.inner-intro a {
+    text-decoration: none;
+    color: var(--paars);
+}
+
+/* --------------- 4.3 Work --------------------------- */
+
+.work-tekst {
+    text-align: left;
+    width: 80%;
+    padding: 2rem;
+}
+
+.image img {
+    width: 100%;
+    box-shadow: 2px 2px 10px 5px rgba(0, 0, 0, 0.2)
+}
+
+.image {
+    flex-basis: auto;
+    width: 100%;
+    flex-grow: 2;
+}
+
+.line {
+    width: 50px;
+    padding-block: 20px;
+}
+
+.work-tekst p {
+    padding-bottom: 20px;
+}
+
+.view-project {
+    color: var(--paars);
+    font-size: 0.7rem;
+    font-weight: bold;
+    text-transform: uppercase;
+    text-decoration: none;
+}
+
+/* --------------- 4.4 Sevices ------------------------ */
+
+.inner-container-services {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 900px;
+    padding: 20px;
+    margin: 2rem;
+}
+
+/* --------------- 4.5 Form --------------------------- */
+
+#contact {
+    background-color: var(--paars);
+    color: #fff;
+    padding: 2rem;
+}
+
+.form {
+    width: 100%;
+}
+
+form {
+    display: flex;
+    flex-direction: column;
+    margin: 2rem;
+}
+
+label {
+    font-weight: bold;
+}
+
+input {
+    padding: 10px;
+    margin-bottom: 20px;
```

Comment:
Ik zie door de hele file dat je op de ene plek px gebruikt en elders rem. Het is fijn om hierin consistent te blijven

Reply:
Thanks voor het nakijken!